src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -313,13 +383,25 @@ public void onStart(ITestContext context) {
                 + Global.WIKI_VERSION + "</p>" + "<div id='toc'></div>");
     CommonUtils.appendTextToFile(logPath, builder.toString());
     appendShowHideButtons();
+
     try {
       FileInputStream input = new FileInputStream("./src/test/resources/script.txt");
       String content = IOUtils.toString(input);
       CommonUtils.appendTextToFile(logPath, content);
     } catch (IOException e) {
       System.out.println("no script.txt file available");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS106)